pkg/rbac/validate_test.go

```diff
@@ -167,3 +167,115 @@ func TestValidateTerms(t *testing.T) {
 		})
 	}
 }
+
+func TestCan(t *testing.T) {
+	t.Parallel()
+	testcases := []struct {
+		request []string
+		can     bool
+	}{
+		{
+			request: []string{
+				"admin",
+				"create",
+				"database-clusters",
+				"*",
+			},
+			can: true,
+		},
+		{
+			request: []string{
+				"admin",
+				"read",
+				"database-clusters",
+				"*",
+			},
+			can: true,
+		},
+		{
+			request: []string{
+				"admin",
+				"update",
+				"database-clusters",
+				"*",
+			},
+			can: true,
+		},
+		{
+			request: []string{
+				"admin",
+				"update",
+				"database-cluster-backups",
+				"*",
+			},
+			can: true,
+		},
+		{
+			request: []string{
+				"alice",
+				"create",
+				"database-clusters",
+				"*",
+			},
+			can: false,
+		},
+		{
+			request: []string{
+				"alice",
+				"read",
+				"database-engines",
+				"*",
+			},
+			can: true,
+		},
+		{
+			request: []string{
+				"alice",
+				"create",
+				"database-clusters",
+				"alice/alice-cluster-1",
+			},
+			can: true,
+		},
+		{
+			request: []string{
+				"bob",
+				"create",
+				"database-clusters",
+				"*",
+			},
+			can: false,
+		},
+		{
+			request: []string{
+				"bob",
+				"create",
+				"database-clusters",
+				"dev/*",
+			},
```

Comment:
from that perspective the `all` alias looks a bit confusing since we can use `all` instead of `*` but we can't use `dev/all` instead of `dev/*` 
I'm not saying we should have the `dev/all` I just wanted to highlight inconsistency

Reply:
Hmm, good point. If you put `*` in the command, it misbehaves, probably some shell thing. But that means it should also work with with `namespace/all` too..

Reply:
regarding the `namespace/all` - what if we have the namespace named `all`?😅 I even saw Yusaf was using that name

well, it might be not such a big deal after all. We can just add more usage examples and/or describe how to use the "all" alias.

Reply:
> what if we have a resource named `all`?

I won't completely rule out the possibility of this, but I think the likelihood of this is very low, given that `all` for a resource type in Everest sounds very vague and may not make much sense 🙈 

But yes, I shall add more examples..

Reply:
Ok, I realised that this `all` alias creates more confusion and increases complexity as well. So I just removed it, and added a note in the example that the user must put quotes around it, otherwise the shell will misinterpret it.. Will also make sure to add it in the docs